DigitalLearningSolutions.Web.Tests/Helpers/ProfessionalRegistrationNumberHelperTests.cs

```diff
@@ -105,25 +105,25 @@ public void ValidateProfessionalRegistrationNumber_sets_error_when_hasPrn_is_not
             using (new AssertionScope())
             {
                 state.IsValid.Should().BeFalse();
-                state.Values.First().Errors.Should().OnlyContain(e => e.ErrorMessage == "Select your professional registration number status.");
+                state.Values.First().Errors.Should().OnlyContain(e => e.ErrorMessage == "Select your professional registration number status");
             }
         }
 
-        [TestCase(null, "Enter professional registration number.")]
-        [TestCase("", "Enter professional registration number.")]
-        [TestCase("123", "Professional registration number must be between 5 and 20 characters.")]
-        [TestCase("0123456789-0123456789", "Professional registration number must be between 5 and 20 characters.")]
+        [TestCase(null, "Enter professional registration number")]
+        [TestCase("", "Enter professional registration number")]
+        [TestCase("123", "Professional registration number must be between 5 and 20 characters")]
+        [TestCase("0123456789-0123456789", "Professional registration number must be between 5 and 20 characters")]
         [TestCase(
             "01234_",
-            "Invalid professional registration number format. Only alphanumeric (a-z, A-Z and 0-9) and hyphens (-) allowed."
+            "Invalid professional registration number format. Only alphanumeric (a-z, A-Z and 0-9) and hyphens (-) allowed"

```

Comment:
Hmm, seems strange to have one sentence with a full stop and one without. Could we replace it with a "-" or a ":"?

Reply:
done